frontend/src/components/UI/Form/CreateBudgetLinesForm.jsx

```diff
@@ -171,6 +171,7 @@ export const CreateBudgetLinesForm = ({
                     </div>
                 ) : (
                     <button
+                        id="add-budget-line"
```

Comment:
learned that the `data-cy` property is the recommended way to target elements for cypress tests.  So here we could do something like `data-cy="add-budget-line-btn"`